packages/loader/container-definitions/src/browserPackage.ts

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IFluidPackageEnvironment, IFluidPackage, isFluidPackage } from "./fluidPackage";
+
+/**
+ * A specific Fluid package environment for browsers
+ */
+export interface IFluidBrowserPackageEnvironment extends IFluidPackageEnvironment{
+    /**
+     * The umd target specifics the scripts necessary for loading a packages
+     * in a browser environment
+     */
+    umd: {
+        /**
+         * The bundled js files for loading this package
+         */
+        files: string[];
+
+        /**
+         * The global name that the script entry points will be exposed.
+         * This entry point should be an IFluidModule
+         */
+        library: string;
+    },
+    /**
+     * The css target optionally specifics the necessary stylesheets for this package
+     */
+    css?: {
+        /**
+         * The css files to include on the page
+         */
+        files: string[];
+    }
+}
+
+/**
+ * A Fluid package for specification for browser environments
+ */
+export interface IFluidBrowserPackage extends IFluidPackage {
+    /**
+     * {@inheritdoc}
+     */
+    fluid: {
+        /**
```

Comment:
Would the interpretation here be "default" value is browser, while any environment specific overrides are in the map? Would a comment here help communicate that?

Reply:
I wouldn't say it is default. If a code loader is in a browser environment, it can use isFluidBrowserPackage  to determine if the fluid package has a browser environment which should then be used. the package could also contain other environments (like node), but those wouldn't be used within a browser